src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Dict, List
+
+
+def load_from_file(file_path: str) -> Dict[str, List[str]]:
+    """
+    This function extracts labels from a Python file.
+    Labels must be in the first line of a Python file and start with "# labels: "
+    Each label must have the format "key::value1,value2,..."
+
+    Example:
+        "# labels: author:google test_group:daily,monthly"
+    """
+    # Labels must be in the first line of the python file
+    with open(file_path, encoding="utf-8") as f:
+        first_line = f.readline()
+
+    if "# labels:" in first_line:
+
+        # Convert labels to dict to ensure they are not malformed
+        label_list = first_line.replace("\n", "").split(" ")[2:]
+        label_dict = convert_to_dict(label_list)
+        return label_dict
+    else:
+        return {}
+
+
+def load_from_cache(cache_dir: str, build_name: str) -> Dict[str, List[str]]:
```

Comment:
This is almost the same function as `load_from_file()`. Can you reduce the code replication by converging them?

Reply:
`load_from_file` loads from a Python file, while `load_from_cache` loads from the cache directory. Since we might change where labels are located in the future it is useful for those functions to have different inputs.

`load_from_file` receives a `file_path` as an input while `load_from_cache` receives `cache_dir` + `build_name`

Here are the options I see:
* Having a single function that receives either `file_path` or `cache_dir` + `build_name` (confusing)
* Forcing the user to insert the path of the cached label (if we change where the labels are stored use must change their code)
* Use the revised code with a bit less overlap I just pushed

Let me know which option you prefer.